packages/auth-construct/src/lambda/sample_data.ts

```diff
@@ -0,0 +1,449 @@
+import { IdentityPool as IdentityPoolType } from '@aws-sdk/client-cognito-identity';
+import {
+  GetUserPoolMfaConfigCommandOutput,
+  ListGroupsResponse,
+  ProviderDescription,
+  UserPoolClientType,
+  UserPoolType,
+} from '@aws-sdk/client-cognito-identity-provider';
+import { ReferenceAuthInitializerProps } from '../types.js';
+
+/**
+ * Sample referenceAuth properties
+ */
+export const SampleInputProperties: ReferenceAuthInitializerProps = {
```

Comment:
we should add this file to `.npmignore` .

perhaps we can have `test_resources` subfolder and add that to `.npmignore`.

Reply:
updated , just used test-resources folder because of kebab linting error for test_resources